database/migrations/2016_07_26_175053_create_reportback_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateReportbackTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('reportback', function (Blueprint $table) {
+            $table->increments('id')->comment('Primary key. Maps to rbid');
+            $table->string('user_id')->comment('The rogue users.id that correlates to their Northstar id');
+            $table->integer('uid')->comment('The phoenix users.uid that reported back.');
+            $table->integer('nid')->comment('The node.nid that the user has reported back for.');
```

Comment:
I'm also wondering about these names, and maybe the crew will have a lot of feelings and can debate this later, but I'd rather go with `node_id` or something. hmmm